Add Feature Flag Migration guide to docs (#4792)
Unleash/unleash: website/docs/topics/feature-flag-migration/business-case-feature-flag-migration.md

> **Make the business case for feature flag migration**

Reviewer: This looks like it should be a title, not bolded text.

```suggestion
---
title: Make the business case for feature flag migration
---
```

---
title: Make the business case for feature flag migration
---

> Once you have scoped your migration, you need to make a business case. Even the most well-planned migrations are work. Work means time, money, and energy dedicated to a project.  If you don’t have the proper buy-in, you risk not being under-resourced or worse, being unable to complete the migration at all.

Reviewer 2: ```suggestion
Once you have scoped your migration, you need to make a business case. Even the most well planned migrations take effort, meaning time, money, and energy dedicated to a project.  If you don’t have the proper buy-in, you risk being under-resourced or worse, being unable to complete the migration at all.
```
